ckanext/unaids/i18n/extra_translations.py

```diff
@@ -48,5 +48,5 @@
     _('A spectrum file used to run the Naomi model'),
     _('License definitions and additional information can be found at http://opendefinition.org/'),
     _('Reorder resources'),
-
+    _('Organisation Name / ID')
```

Comment:
The string "Username / Email" doesn't need to be added as a string here because it is referenced directly from the code in other templates and will be picked up from there.  Of course I could add it in here for completeness if this is deemed important.